pkg/provider/loadbalancerclass_test.go

```diff
@@ -2,6 +2,8 @@ package provider
 
 import (
 	"context"
+	"k8s.io/api/core/v1"
+	core "k8s.io/client-go/testing"
```

Comment:
nit: I noticed https://github.com/kubernetes/sample-controller/blob/master/controller_test.go#L33 it used core, but it's kind of confusing against k8s core api, maybe let's use `clientgotesting`